docs/en/03-tx-sharing.md

```diff
@@ -2,30 +2,31 @@
 title: Transaction Sharing
 ---
 
-A client can send a transaction to any of the Ostracon nodes that joining the blockchain network. The transaction propagates to other Ostracon nodes and is ultimately shared by all Ostracon nodes.
+A client application can send a transaction to any Ostracon nodes that join the blockchain network. The transaction propagates to other Ostracon nodes and is ultimately shared by all Ostracon nodes.
 
 ## Mempool
 
-Once a block is accepted by the Ostracon consensus mechanism, the transactions contained in that block are considered *confirmed*. The unconfirmed transactions are validated stored in an area called [**mempool**](https://github.com/tendermint/tendermint/blob/v0.34.x/spec/abci/apps.md#mempool-connection), which is separate from the block storage, after validation such as signatures.
+<!-- Once a block is accepted by the Ostracon consensus mechanism, the transactions contained in that block are considered **confirmed**.  -->
```

Comment:
Did you forget to remove it?

Reply:
The commented parts are those that were so in the LBM document. It's removed from this document.